src/formio/components/currency.ts

```diff
@@ -0,0 +1,29 @@
+import {InputComponentSchema, MultipleCapable, PrefillConfig} from '..';
+
+type Validator = 'required' | 'min' | 'max';
+type TranslatableKeys = 'label' | 'description' | 'tooltip' | 'suffix';
```

Comment:
```suggestion
type TranslatableKeys = 'label' | 'description' | 'tooltip';
```
suffix is not used with currency component atm